packages/utils/src/libp2p/dialHelper.ts

```diff
@@ -25,7 +25,7 @@ const CODE_P2P = maProtocols('p2p').code
 const log = debug(DEBUG_PREFIX)
 const error = debug(DEBUG_PREFIX.concat(`:error`))
 
-const DEFAULT_DHT_QUERY_TIMEOUT = 20000
+const DEFAULT_DHT_QUERY_TIMEOUT = 15000
```

Comment:
This should go into the overall config preferably.

Reply:
Need to again split the timeout in a part for establishing the connection and one for interacting with the protocol. Will most likely do this in a follow-up PR